### PR TITLE
Add man pages for ocitools

### PR DESCRIPTION
**Pull request by rhatdan**
Also fix up Makefile so that it will install ocitools and man pages.

### CODE REVIEW

File: man/ocitools-generate.1.md [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+
+Then exit and check the journal.
+
+    # exit
+
+    # journalctl -b | grep Hello
+
+This should list the message sent to logger.
+
+## Attaching to one or more from STDIN, STDOUT, STDERR
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: This does not exist in ocitools.

File: man/ocitools-generate.1.md [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+```
+ $ ocitools generate -it --ipc=container:ed735b2264ac shm ipcs -m
+ $ sudo ipcs -m
+
+ ------ Shared Memory Segments --------
+ key        shmid      owner      perms      bytes      nattch     status
+ 0x0000162e 0          root       666        27         1
+```
+
+## Mapping Ports for External Usage
```
Comment (mrunalp) [inline man/ocitools-generate.1.md]: Also, ports have to be handled at a higher level. We can't manage them in ocitools.

Comment (rhatdan) [inline man/ocitools-generate.1.md]: I was just cutting and pasting docker-run man page.  On an early Saturday/Sunday morning. Will remove these sections.

**Comment by rhatdan** [comment]
Fixed up the SELInux example.

**Comment by rhatdan** [comment]
Removed the bogus examples.

BTW Why did you call it bvalidate rather then validate?

**Comment by mrunalp** [comment]
I think we can rename it.

**Pull request closed without merging**